src/gimelstudio/core/project_file.py

```diff
@@ -144,8 +148,11 @@ def WriteFile(self, file_path, contents):
 
     def OpenFile(self, file_path):
         with open(file_path, "r") as file:
-            file_contents = json.load(file)
-            self.contents = file_contents
+            try:
+                file_contents = json.load(file)
+                self.contents = file_contents
+            except Exception as e:
```

Comment:
I don't think there is much point having `Exception as e` here as you aren't using the message itself.
```suggestion
            except:
```